engine/classes/Elgg/Database/MetadataTable.php

```diff
@@ -41,6 +41,9 @@ class MetadataTable {
 	
 	/** @var string */
 	protected $table;
+	
+	//this constant checks that column size is not exceeded
```

Comment:
Constants don't check anything. Better would be to name the constant more descriptively so you don't need a comment: `MYSQL_TEXT_BYTE_LIMIT`